functorch/csrc/DynamicLayer.cpp

```diff
@@ -594,6 +637,8 @@ void dynamicLayerBackFallback(const c10::OperatorHandle& op, torch::jit::Stack*
     }
     // Step 2
     foreachTensorInplace(*stack, stack->size() - args_size, stack->size(), unwrap);
+  } else if (cur_key == DispatchKey::Functionalize) {
+    sanityCheckNotFunctional(op, stack, args_size);
```

Comment:
Makes sense, but we'll need to change this for nested functionalize, right? (if that's a correct assessment then a comment here would be a nice reminder)

Reply:
yes, and agreed - i'll add a comment to make it clear that this should be taken out when we eventually have nested functionalization support (I have it there mostly for sanity to help with debugging)